host/libhackrf/src/hackrf.c

```diff
@@ -724,6 +727,8 @@ static int hackrf_open_setup(libusb_device_handle* usb_device, hackrf_device** d
 	lib_device->active_transfers = 0;
 	lib_device->flush = false;
 	lib_device->flush_transfer = NULL;
+	lib_device->flush_callback = NULL;
+	lib_device->flush_ctx = NULL;
```

Comment:
Should this also set do: `lib_device->tx_completion_callback = NULL;`?

Reply:
Fixed.